source/client-side-encryption/tests/maxWireVersion.template.yml

```diff
@@ -0,0 +1,20 @@
+runOn:
+  - maxServerVersion: "4.0.0"
```

Comment:
This should be just `4.0`.  Otherwise I can see implementations interpreting that it shouldn't run on a `4.0.10` server, for example.  Alternatively it could be `4.0.<maxInteger>`, but that just seems weird.

Reply:
Done.